discord/enums.py

```diff
@@ -418,6 +418,22 @@ class PremiumType(Enum):
     nitro_classic = 1
     nitro = 2
 
+class ExpireBehaviour(Enum):
+    remove_role = 0
+    kick = 1
+
+class ExpireGracePeriod(Enum):
```

Comment:
Personally I don't like this enum. It's pretty useless. An integer with a documentation saying what the unit is would be much more user friendly.